examples/b2c/checkout/integrations/classes/B2CDeliverySample.cls-meta.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
-    <apiVersion>55.0</apiVersion>
+    <apiVersion>50.0</apiVersion>
```

Comment:
B2C class is never used. Only B2B classes are used and since it is renamed you are seeing the difference. We didn't make any change to the classes or xml files